### PR TITLE
Fixup demo

### PR DESCRIPTION
**Pull request by kryachkov**

### CODE REVIEW

File: app/controllers/v1/base_controller.rb [hunk 1]
```diff
@@ -82,7 +82,6 @@ def token_should_be_refresh
   end
 
   def set_time_zone
-    timezone = ActiveSupport::TimeZone[params["timezone_offset"].to_i]
-    Time.use_zone(timezone) { yield }
```
Comment (x13enos) [inline app/controllers/v1/base_controller.rb:86]: test

Comment (kryachkov) [reply app/controllers/v1/base_controller.rb:86]: Makes sense, please see c246e2b5dfccf2d2a6c3a589fd03e117d158b4a4

File: app/controllers/v1/base_controller.rb [hunk 2]
```diff
@@ -82,7 +82,6 @@ def token_should_be_refresh
...
+    Time.use_zone(params["current_timezone"]) { yield }
```
Comment (volonterx) [inline app/controllers/v1/base_controller.rb]: Test

Comment (kryachkov) [reply app/controllers/v1/base_controller.rb]: Makes sense. Please see d738acb

**Pull request closed without merging**